setup.py

```diff
@@ -62,5 +62,5 @@
     entry_points={
         'console_scripts': ['cortexpy=cortexpy.__main__:main'],
     },
-    include_package_data=True,
+    include_package_data=True, install_requires=['numpy'], install_requires=['numpy']
```

Comment:
This is a syntax error (https://travis-ci.org/winni2k/cortexpy/jobs/431234519#L843). There is already an install_requires farther up on line 36.